client/js/components/map/map/DpOlMap.vue

```diff
@@ -391,6 +390,7 @@ export default {
       const extent = view.getProjection().getExtent()
       const center = view.getCenter()
 
+      // Force a recalculation of the map viewport size. This should be called when third-party code changes the size of the map viewport.
```

Comment:
You can remove this comment because it is kind of redundant. You can see the exact same explanation when hovering over the function in your IDE

Reply:
done https://github.com/demos-europe/demosplan-core/pull/1413/commits/acb630022b689454783c7873a9a770d435850d3d